### PR TITLE
Add contribution guide and links to readme

### PR DESCRIPTION
**Pull request by vasarhelyia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+It's recommended to install the [Bitrise CLI](https://www.bitrise.io/cli) on your machine so you can test your step locally.
+
+You can install it via Homebrew:
+
+`brew install bitrise`
+
+Or check the latest release with instructions at: [https://github.com/bitrise-io/bitrise/releases](https://github.com/bitrise-io/bitrise/releases)
+
+### Share your step
```
Comment (viktorbenei) [inline README.md]: Share guide & example: https://github.com/bitrise-steplib/step-template

Comment (vasarhelyia) [inline README.md]: added

Comment (viktorbenei) [inline README.md]: where? ;)

Comment (vasarhelyia) [inline README.md]: https://github.com/bitrise-io/bitrise-steplib/pull/299/files#diff-04c6e90faac2675aa89e2176d2eec7d8R7

**Pull request merged**